pkg/utils/eden.go

```diff
@@ -0,0 +1,117 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	adamContainerName = "eden_adam"
+	adamContainerRef  = "lfedge/adam"
+)
+
+//StartAdam function run adam in docker with mounted adamPath/run:/adam/run
+//if adamForce is set, it recreates container
+func StartAdam(adamPort string, adamPath string, adamForce bool) (err error) {
+	if adamForce {
+		err = StopContainer(adamContainerName, true)
```

Comment:
Nitpick, but this would be cleaner (and more standard) as:

```go
if err := StopContainer(adamContainerName, true); err != nil {
    return fmt.Errorf("error in rm Adam container: %s", err)
}
```

Reply:
I refactored the code, thanks.